_layouts/home.html

```diff
@@ -22,5 +22,12 @@ <h1 class="page-heading center">Contributors</h1>
         <i class="fa fa-quora"></i>
       </a>
     </li>
+    <li>
+      <a href="https://github.com/sushant-j/"> 
+      Ganesh Chaudhari
+      </a>
+      
```

Comment:
Hey @Ganesh-chaudhari  - Could you remove these blank lines please? 
I'd happy to remove it from Github UI if not.